src/styles/styles.js

```diff
@@ -2194,6 +2196,10 @@ const styles = {
         marginBottom: -10,
     },
 
+    iouPreviewBoxAvatarHover: {
+        borderColor: themeColors.activeComponentBG,
```

Comment:
**Not a blocker**: I think we should use a border variable here for easy customizability.